modules/apps/asset/asset-taglib/src/main/resources/META-INF/resources/asset_tags_selector/AssetTagsSelector.es.js

```
@@ -159,17 +162,7 @@ class AssetTagsSelector extends Component {
 		this.selectedItems = event.data.selectedItems;
 		this.tagNames = this._getTagNames();
 
-		if (this.removeCallback) {
-			window[this.removeCallback](event.data.item);
-		}
-
-		this.emit(
-			'itemRemoved',
-			{
-				item: event.data.item,
-				selectedItems: this.selectedItems
-			}
-		);
+		this._notifyItemsChanged('itemRemoved', this.removeCallback, item);
```

`item` is not defined here. This should throw an exception.